packages/netlify-cms-lib-util/src/unsentRequest.js

```diff
@@ -88,6 +88,7 @@ const getPropSetFunction = path => getCurriedRequestProcessor((val, req) => req.
 const getPropMergeFunction = path =>
   getCurriedRequestProcessor((obj, req) => req.updateIn(path, (p = Map()) => p.merge(obj)));
 
+const withCredentials = getPropSetFunction(['credentials']);
```

Comment:
Since we are always calling `withCredentials` using `include` we can use:

```suggestion
const withIncludeCredentials = getPropSetFunction(['credentials'])('include');
```

Directly, WDTY?

Reply:
In the future someone might want to use `same-origin` instead of `include`.

Reply:
> In the future someone might want to use `same-origin` instead of `include`.

We can update the code in the future to support this if needed.
As the current API doesn't let you set the `credentials` value anyway, it's better to be as specific as possible.